apps/examples/testcase/le_tc/compression/tc_compress_read.c

```diff
@@ -56,12 +126,48 @@ static void tc_compress_decompress(void)
 	TC_SUCCESS_RESULT();
 }
 
+static void tc_comp_decomp_buffer(void)
+{
+	int fd;
+	int ret_chk;
+	fd = get_compfd();
+
+	/*If FAIL : Failed to open testcase driver*/
+	TC_ASSERT_GEQ("get_compfd", fd, OK);
+	
+	// initializing struct to store compressed and decompressed data
+	ret_chk = initialize_tc_data();
+	TC_ASSERT_EQ("compression data initialize", ret_chk, OK);
+
+	// Test to check compression type 
+	ret_chk = ioctl(fd, COMPIOC_GET_COMP_TYPE, comp_tc_data);
```

Comment:
To get comp type, you don't need to give the arg. Is it right?
If yes, why do you give the comp_tc_data as the arg?